src/scss/includes/search_form.scss

```diff
@@ -302,8 +302,11 @@ input[type="search"] {
 
   .autocomplete_suggestions {
     position: fixed;
-    max-height: calc(100vh - 120px);
+    top: 0;
+    height: calc(100vh - 69px);
```

Comment:
This `69px` seems out of nowhere (yes, 120px was too :)). And it doesn't seem right also, I see some pixels of space at the bottom:
![Capture d’écran de 2020-05-15 15-21-49](https://user-images.githubusercontent.com/243653/82055506-e3535d00-96c0-11ea-9b9d-466547a2257e.png)

Reply:
Actually, you can just replace this rule by `bottom: 0;`, and you won't need any magic number :)

Reply:
You resolved this comment without actually addressing it.

Reply:
sorry, my fix the push didn't work :( 
That and my laptop that crashed with a black screen of death earlier, no doubt, it's monday :D